mlabs/src/Mlabs/EfficientNFT/Types.hs

```diff
@@ -565,7 +565,8 @@ data MintCnftParams = MintCnftParams
   deriving anyclass (FromJSON, ToJSON, ToSchema)
 
 data SeabugMetadata = SeabugMetadata
-  { sm'mintPolicy :: ScriptHash
+  { sm'policyId :: ScriptHash -- ^ applied script hash
+  , sm'mintPolicy :: ScriptHash -- ^ not-applied script hash
```

Comment:
we confirmed with Maks that `mintPolicy` is just an arbitrary string, used for versioning. as such, it can be hard coded to `V1`.

Reply:
Fixed.